pgstore.go

```diff
@@ -63,10 +63,10 @@ func NewPGStoreFromPool(db *sql.DB, keyPairs ...[]byte) *PGStore {
 
 	if err != nil {
 		// Ignore and return nil
-		return nil
+		return nil, err
```

Comment:
This return (WITHOUT an error) is KILLING me atm. Please consider merging this PR ASAP.....